tests/debugger_tests/setup-debuggertets.cmd

```diff
@@ -0,0 +1,92 @@
+@if not defined __echo @echo off

```

Comment:
probably want to name this 'setup-debuggertests.cmd' (there is a missing s in there)